models/user.go

```diff
@@ -1,14 +1,15 @@
 //
-// Copyright (C) 2022 IOTech Ltd
+// Copyright (C) 2022-2024 IOTech Ltd
 //
 // SPDX-License-Identifier: Apache-2.0
 
 package models
 
 type User struct {
 	DBTimestamp
-	Id        string
-	Name      string
-	Group     string
-	PublicKey string
```

Comment:
The original `User` model was used in some user db methods in redis  https://github.com/IOTechSystems/edgex-go-private/blob/87ddfb8f3b1585e3f7c86cb2032f11933277d5c6/internal/pkg/infrastructure/redis/client.go#L1752,
but those db methods were not actually being used in the application layer, so should be safe to modify the fields.